File: abc/WhenIsWeekend/Lambda.java
import java.util.Calendar;

public class Lambda {

    public static void main(String[] args) {
        System.out.println("When’s Weekend?");

        int today = Calendar.getInstance().get(Calendar.DAY_OF_WEEK);

        switch (today) {
            case Calendar.SUNDAY:
                System.out.println("Today.");
                break;
            case Calendar.SATURDAY:
                System.out.println("Today.");
                break;
            case Calendar.FRIDAY:
                System.out.println("Tomorrow.");
                break;
            case Calendar.THURSDAY:
                System.out.println("In two days.");
                break;
            default:
                System.out.println("Too far away.");
        }
    }
}

File: abc/WhenIsWeekend/Lambda.js
"use strict"

if (module === require.main) {
    console.log("When’s Weekend?")

    let today = new Date().getDay()

    switch (today) {
        case 0:
            console.log("Today.")
            break
        case 6:
            console.log("Today.")
            break
        case 5:
            console.log("Tomorrow.")
            break
        case 4:
            console.log("In two days.")
            break
        default:
            console.log("Too far away.")
    }
}
